sdk/confidentialledger/Azure.Security.ConfidentialLedger/README.md

```diff
@@ -71,22 +71,17 @@ Since Azure confidential ledger is a distributed system, rare transient failures
 
 ```C# Snippet:GetStatus
 Response statusResponse = ledgerClient.GetTransactionStatus(transactionId);
-
-string status = JsonDocument.Parse(statusResponse.Content)
-    .RootElement
-    .GetProperty("state")
-    .GetString();
+dynamic transactionStatus = statusResponse.Content.ToDynamic();
+string status = transactionStatus.state;
 
 Console.WriteLine($"Transaction status: {status}");
 
 // Wait for the entry to be committed
 while (status == "Pending")
 {
     statusResponse = ledgerClient.GetTransactionStatus(transactionId);
-    status = JsonDocument.Parse(statusResponse.Content)
-        .RootElement
-        .GetProperty("state")
-        .GetString();
+    transactionStatus = statusResponse.Content.ToDynamic();
+    status = transactionStatus.state;
```

Comment:
@annelo-msft, remind me what we decided about allowing pascal casing for dynamic property access.

Reply:
I am working on that in a separate branch now.

We can't implement it with full case-insensitivity as we did before, since we're not storing properties in a Dictionary anymore, where we could provide a case-insensitive Comparer for lookups.  JsonElement.GetProperty() is case sensitive (as far as I can tell), so we will need to proceed with a double-lookup on both PascalCase and camelCase.  I was working on the design for how to configure that.  It has implications for how we call Set on the properties as well.  If they are there, I think the logic is the same - you can access them either PascalCase or camelCase and set them without changing the casing of the property name.  When new properties get added is where it is tricky - ideally you would already know what the casing of the underlying JSON is and do that regardless of how the dynamic member is accessed, but I haven't come up with a good way of determining this yet either.  We will need to decide on defaults as well.

I'm hoping to share a proposal for this tomorrow or early next week, but happy to take suggestions.

Reply:
Oh, this sounds complicated. Maybe we don't want to do it after all. Yeah, let's talk about pros and cons tomorrow.